import_specifications/clients/baseclient.py

```diff
@@ -51,8 +51,7 @@ def _get_token(user_id, password, auth_svc):
         tok = _json.loads(ret.text)
     elif status == 403:
         raise Exception(
-            "Authentication failed: Bad user_id/password "
-            + "combination for user %s" % (user_id)
+            "Authentication failed: Bad user_id/password " + "combination for user %s" % (user_id)
```

Comment:
if you write this as
```py
f"Authentication failed: Bad user_id/password combination for user {user_id}"
```
does it break the line length rule?

Reply:
I have not checked, but I would prefer to leave this autogenerated file untouched/ignored.